stemcell_builder/stages/bosh_users/apply.sh

```diff
@@ -18,8 +18,8 @@ run_in_chroot $chroot "
 groupadd --system admin
 useradd -m --comment 'BOSH System User' vcap
 chmod 755 ~vcap
-echo \"vcap:${bosh_users_password}\" | chpasswd
-echo \"root:${bosh_users_password}\" | chpasswd
+echo "vcap:${bosh_users_password}" | chpasswd
```

Comment:
I tried changing password manually with escape character on Ubuntu trusty and it did not work.  Looks like this code is never seemed to be working correctly.